src/features/rewards/mobile/mainToggleMobile/style.ts

```diff
@@ -61,6 +62,10 @@ export const StyledTM = styled<{}, 'span'>('span')`
   position: relative;
   top: -13px;
   vertical-align: text-top;
+
+  @media (max-width: 375px) {
```

Comment:
Instead of using negative margin, desktop uses flex alignment to get aligned to the top of the box around the text - could that simplify this?